packages/utils/src/webpack.ts

```diff
@@ -12,6 +12,16 @@ export class WebpackConfigurator {
     return this.config;
   }
 
+  public setEntry(name: string, entry: string | string[]): this {
```

Comment:
This function should set entry to a specific value, it shouldn't append to existing one.